slangpy/core/struct.py

```
@@ -17,11 +18,9 @@ class Struct:
     def __init__(
         self, module: "Module", slang_struct: "SlangType", options: dict[str, Any] = {}
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)